Primera pull request del maquetado

Mipof opened the pull request:
En teoria es la pagina del menu de las configuraciones del perfil.

Mipof committed:
html-prueba

Mipof committed:
creacion barra buscar[estetico]

creando y acomodando la barra buscar con flexbox. sin ninguna funcion.

Mipof committed:
ignorar

solo estoy moviendo archivos entre mis dispositivos.

Mipof committed:
se añadio sub menus y "logo" [estetico]

Mipof committed:
se añadio menu seccion cuenta con opciones [estetico]

se agrego pedidos, configuracion, pagos, con algunas opciones extras [estetico]

Mipof committed:
se elimino coigo de pruebas [codigo como comentario]

Mipof committed:
se modificaron fuentes, se añadio footer y cambios esteticos menores

nmicht commented on main.css:
```diff
@@ -0,0 +1,111 @@
+	
+	display: flex;
+	order: 2;
+	flex:3;
```
espacio después de los dos puntos. En varios lugares esta así

nmicht commented on main.css:
```diff
@@ -0,0 +1,111 @@
+}
+
+.pie_pagina{
+	
```
Evitar lineas vacias

nmicht commented on main.html:
```diff
@@ -0,0 +1,73 @@
+			<h1>Logo</h1>
+			<h2>Carrito</h2>
+			<div class="seccion_busqueda">
+					<div class="busqueda">
```
Veo tabulaciones de mas

nmicht commented on main.html:
```diff
@@ -0,0 +1,73 @@
+			<h2>Carrito</h2>
+			<div class="seccion_busqueda">
+					<div class="busqueda">
+						<label for="buscar_label">Buscar:</label>					
```
buscar_label no es un id dentro del sitio

nmicht commented on main.html:
```diff
@@ -0,0 +1,73 @@
+			</div>
+		</header>
+		<div class="menusitos">
+			<h3>menu</h3>
```
No se usan h3 a menos que si exista un h1 y h2 como titulos superiores a esto

nmicht commented on main.html:
```diff
@@ -0,0 +1,73 @@
+		</article>
+		<footer class="pie_pagina">
+				<div class="informacion_general">
+					<h6>Contacto: correo@mercadopirata.com</h6>
```
Revisa el uso de la etiqueta address para datos de contacto

nmicht commented on main.html:
```diff
@@ -0,0 +1,73 @@
+					</div>			
+			</div>
+		</header>
+		<div class="menusitos">
```
Los menus deben ir en la etiqueta nav en lugar de un div

nmicht commented on main.html:
```diff
@@ -0,0 +1,73 @@
+			<h3>categorias</h3>
+			<h3>cuenta</h3>	
+		</div>
+		<article class="menu_cuenta">
```
Creo que esto debería ser mas bien un section

nmicht reviewed (changes requested):
En general hay varios detalles con maquetado y CSS.
Ademas se recomienda que el css este en un folder llamado css

Mipof committed:
modificacion del pull request

Mipof committed:
Modificacion de bordes, nombres y add ventana modificacion

se modificaron los brodes a unos mas sencillos y limpios,
se cambiaron los nombres a unos mas apropidados y se añadio
la ventana de modificacion de informacion del dueño de la cuenta
